plugin.go

```diff
@@ -80,6 +87,53 @@ func (p Plugin) Exec() error {
 
 	return nil
 }
+func customCertHandler(certPath string) *exec.Cmd {
```

Comment:
create a new package "cert" to move related functions into?

Reply:
i think moving to a new file is a bit too much. we are not using this functions from anywhere else. So it doesn't make much sense to me.